qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -241,14 +243,14 @@ class MapboxNavigationViewCustomizedActivity : AppCompatActivity() {
             routeArrowOptions = this@MapboxNavigationViewCustomizedActivity.routeArrowOptions
         }
 
-        // This is not the intended long term solution. We have not yet decided how
-        // to expose view model actions. Considering that we need a navigationView to be created
-        // before we can interact with the view models, we will probably prefer something like this:
-        //       binding.navigationView.api.enableReplay();
-        // TODO Make a ticket with link to list of public api needs
-        binding.toggleReplay.isChecked = binding.navigationView.isReplayEnabled
+        // TODO temporary access to SharedApp.state. Stable api support for this is coming.
+        binding.toggleReplay.isChecked = SharedApp.state.tripSession.isReplayEnabled
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/pull/5922

Not sure if this state is being exposed 🤔 

cc: @tomaszrybakiewicz

Reply:
Feel like this is also temporary, but made a function on NavigationView
```
    /**
     * Check if the current trip is being simulated.
     */
    fun isReplayEnabled(): Boolean {
        return navigationContext.store.state.value.tripSession.isReplayEnabled
    }
```